.github/workflows/verible-ci.yml

```diff
@@ -424,7 +424,7 @@ jobs:
           name: bazel-bin
 
       - name: Extract bazel-bin
-        run: tar xvf bazel-bin.tar 
+        run: sudo tar xvf bazel-bin.tar -C /usr/bin 
```

Comment:
I'd probably not unpack them somewhere in the public /usr/bin, but somewhere local as we don't want the the regular path get into the way (e.g. `/tmp/verible-bin`) Then use that path to invoke `github-pages-setup.sh /tmp/verible-bin`

Also, you don't need 'sudo', then :)

Reply:
Done.